ebuild-maintenance/git/text.xml

```diff
@@ -324,6 +323,98 @@ Closes: https://bugs.gentoo.org/00001
 Signed-off-by: Alice Bar &lt;a.bar@example.org&gt;
 </pre>
 
+</body>
+</subsection>
+
+<subsection>
+<title>Certificate of Origin</title>
+<body>
+
+<p>
+Per Gentoo's <uri link="https://www.gentoo.org/glep/glep-0076.html">Copyright
+Policy GLEP</uri>, you must sign-off all your commits to any Gentoo-hosted
```

Comment:
maybe Copyright Policy (GLEP 76)?

Reply:
Added `Per Gentoo's <uri link="https://www.gentoo.org/glep/glep-0076.html">GLEP 76
(Copyright Policy GLEP)</uri>` since it's the first thing that came to my mind after reading your first comment, before reading your 2nd :)